cypress/fixtures/analysis.json

```diff
@@ -54,30 +61,15 @@
         "storyPoints": 42
     },
 
-    {
+    "Tackle-testapp-exclude-packages": {
         "source": "Source code",
```

Comment:
For this as well same tackle-testapp can be used.

Reply:
Here we have provided the ``` "excludePackages": "io.konveyor.demo.ordermanagement.model", ``` if we try to run with tackle-testapp all the TCs by default will run with ```excludePackages```